meta/change_management.md

```diff
@@ -0,0 +1,65 @@
+# Change Management Process
+
+This document describes the change management process for the rule book and all related content and meta files written in Markdown in this project. The process will change based on what phase of the initial project we're in at the moment. Each phase is described below.
+
+Note that changes to this document are also governed by the process described in this document.
+
+## Phase 1: Version 1 Development
+
+The initial version development of this project consists of all work conducted on or after Sunday, November 1, 2020 through to the beginning of phase 2, as defined by phase 2. There are 4 developers involved in phase 1:
+
+- Gryphon Shafer
+- Scott Peterson
+- Jeremy Swingle
+- Zachary Tinker
+
+During the initial version development of this project, all changes will be made as follows:
+
+1. A change's author should submit the change as a *Pull Request* (PR) to the project's `master` branch
+2. At least 1 developer other than the author should comment on the PR; however, ideally many developers will comment
+3. Discussion, debate, and revisions to the PR happen all within GitHub's PR system
+4. The PR is merged to `master` when any of the following is true:
+    - 1 developer other than the author agrees with the change and there are no disagreements from the other developers after 1 week from PR submission
+    - 2 developers other than the author agree with the change
+
+## Phase 2: Version 1 Quality Assurance
+
+Phase 2 begins once all 4 developers agree that all primary writing and editing as part of phase 1 has concluded. Additional editing may still take place, but at this point, all 4 developers believe they have a working product that, while potentially containing errors, is not likely missing information.
+
+The following are the steps of phase 2:
+
+1. Gryphon conducts an "OCD pass" of all content, making it align exactly with syntax, style, and other requirements
+2. Gryphon submits a PR for review by the other 3 developers
+3. Discussion and debate take place as part phase 1 PRs, and eventually the PR is merged to `master`
+4. Gryphon builds a single *Adobe PDF file* (PDF) for the entire project
+5. Gryphon creates a PR covering the entire project
+
+## Phase 3: CQLT Review and Approval
+
+The PDF will be submitted to the CQLT for review and approval. The 6 quizzing members of the CQLT will have 1 calendar month from the date of submission to review and provide feedback on the submission. Any feedback other than general approval should be made as GitHub comments to the PR covering the entire project.
+
+The 4 developers will respond to any feedback, potentially creating subsequent versions of the output from phase 2 as necessary.
+
+At the end of the 1 month review period of phase 3, if there are no unresolved comments regarding *functional equivalence* (FE) variance, the 6 quizzing members of the CQLT will vote to approve.
```

Comment:
I think this is plenty explicit, but my fears of random and unexpected LIM veto hammer persists.

Reply:
LIM doesn't get a vote as to decisions made by the CQLT. As written, our bylaws should prevent that from even being a thought. LIM is to provide logistical support as paid for by the CQLT. Anything outside of that would be outside the scope of LIM's involvement.

Reply:
Was Internetionals not being sponsored by the CQLT a 6-quizzing-member CQLT decision?

Reply:
Ultimately, yes. LIM and Alliance Youth can give suggestions, however their two positions are considered non-voting according to the bylaws.

Reply:
According to the bylaws the only things the LIFE Office can do are these three things. 

3. Life Office
The Life Office shall provide logistical support for the International Quizzing
Competition and the director will provide operational oversight of the event.
A. The LIFE Office shall recommend venues for the international competition to the CQLT for approval.
B. The LIFE Office shall coordinate the international quizzing event and act as the sole liaison between the CQLT and the staff of the venue.
C. The LIFE Office will provide bookkeeping services, including billing and paying expenses for the CQLT as requested.

4. Budget and Finance
The CQLT shall prepare a balanced annual programming budget including the International Quizzing event, operational expenses and fees for LIFE Office services. A copy of the budget shall be forwarded to the Vice President Nurture & Development in Canada and the Vice President for U.S. Church Ministries for their information. The CQLT shall provide annual financial reports to each C&MA national office and each district quizzing program.

5. Amenability
The CQLT and LIFE office is amenable to the Vice President for Church Ministries in the U.S.